src/test/kotlin/ar/edu/unahur/obj2/caralibro/UsuarioTest.kt

```diff
@@ -20,15 +117,125 @@ class UsuarioTest : DescribeSpec({
           saludoCumpleanios.espacioQueOcupa().shouldBe(45)
         }
       }
+
+      describe ("de tipo Video") {
+        it ("cuanto espacio ocupa el SD") {
+          videoDeCumpleanios.espacioQueOcupa().shouldBe(3000)
+        }
+        it ("cuanto espacio ocupa el HD720") {
+          videoDeCumpleaniosEn720.espacioQueOcupa().shouldBe(9000)
+        }
+        it ("cuanto espacio ocupa el HD1080") {
+          videoDeCumpleaniosEn1080.espacioQueOcupa().shouldBe(18000)
+        }
+        videoDeCumpleanios.cambiarCalidadDelVideo("HD720")
+        it ("cuanto espacio ocupa el SD despues de cambiar su calidad a HD720") {
+          videoDeCumpleanios.espacioQueOcupa().shouldBe(9000)
+        }
```

Comment:
Si la acción solo tiene sentido dentro de este test, conviene moverla adentro. Así como está puede afectar a otros tests:

```suggestion
        it ("cuanto espacio ocupa el SD despues de cambiar su calidad a HD720") {
          videoDeCumpleanios.cambiarCalidadDelVideo("HD720")
          videoDeCumpleanios.espacioQueOcupa().shouldBe(9000)
        }
```